VideoLocker/res/values/styles.xml

```diff
@@ -106,6 +105,10 @@
         <item name="android:background">@color/edx_brand_primary_base</item>
     </style>
 
+    <style name="course_dashboard_separator_style" parent="separator_style">
```

Comment:
How many separator styles do we have? Can we just get away with "light" and "dark" or something?